DATA/books.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
- [ ] Your linters are not passing. Kindly make your linters pass all the tests because linter prevent you making those errors which can break your application :thinking: